faults.md

```diff
@@ -14,39 +14,81 @@ A fault is what happens when partcipants in the protocol are behaving incorrectl
-- **Reported storage fault penalty:**
-  - **Condition:** The miner submits their PoSt with a non-empty set of 'missing sectors'.
-  - **Reporting:** The miner can specify some sectors that they failed to prove during the proving period.
-    - Note: These faults are output by the `ProveStorage` routine, and are posted on-chain when posting the proof. This occurs when the miner (for example) has a disk failure, or other local data corruption.
-  - **Check:** The chain checks that the proof verifies with the missing sectors.
-  - **Penalization:** The miner is penalized for collateral and power proportional to the number of missing sectors. The sectors are also removed from the miners proving set.
-    - TODO: should the collateral lost here be proportional to the remaining time?
-    - TODO(nicola): check if the time between posting two proofs allows for a generation attack if it does not then we might reconsider the sector not being lost
-  - Note: if a sector is missed here, and they are recovered after the fact, the miner could simple 're-commit' the sector. They still have to pay the collateral, but the data can be quickly re-introduced into the system to avoid clients calling them out for breach of contract (this would only work because the sector commD/commR is the same)
-  - Note: In the case where a miner is temporarily unable to prove some of their data, they can simply wait for the temporary unavailability to recover, and then continue proving, submitting the proofs a bit late if necessary (paying appropriate fees, as described above).
-- **Breach of contract dispute:**
-  - **Condition:** A client who has stored data with a miner, and the miner removes the sector containing that data before the end of the agreed upon time period.
-  - **Reporting:** The client invokes `ArbitrateDeal` on the offending miner actor with a signed deal from that miner for the storage in question. Note: the reporting must happen within one proving period of the miner removing the storage erroneously.
-  - **Check:** The chain checks that the deal was correctly signed by the miner in question, that the deal has not yet expired, and that the sector referenced by the deal is no longer in the miners proving set.
-  - **Penalization:** The miner is penalized an amount proportional to the incorrectly removed sector. This penalty is taken from their pledged collateral .
-  - Note: This implies that miners cannot re-seal data into different sectors. We could come up with a protocol where the client gives the miner explicit consent to re-seal, but that is more complicated and can be done later.
+
+**Definitions**
+
+- `PoStProvingPeriod: UInt = 24 * 60 * 60 / 30 = 2880` (1 day, given `30s` block time)
+- `PoStTimeout: UInt` (less than 1 ProvingPeriod)
+- `SectorFailureTimout: UInt` (more than 1 Proving Period)
+
```

Comment:
The generation attack time will go away entirely in these constructions. It will be only used to reason about these timeouts when defining them, but not appear in the spec in the current construction. 

This leaves us with following periods

- on time
- between PPE and PoStTimeout
- between PoStTimeout and SectorFailureTimeout
- no submission/submission after SectorFailureTimeout